test/integration/examples/http/http.go

```diff
@@ -650,6 +653,73 @@ var h1HeyBaseJob = &batchv1.Job{
 	},
 }
 
+// This is required to avoid test flakiness while #1208 is not fixed.  It will wait for the
+// secret to be created up to a maximum time.
+//
+// It returns nothing and ignores any errors that occur, leaving any inconsistencies to be
+// detected later on by other code
+func waitSecret(cctx *base.ClusterContext, secretName string) {
+	// The check below is required while https://github.com/skupperproject/skupper/issues/1208
+	// is not fixed; both the ServiceInterfaceCreate above and ServiceInterfaceBind below may
+	// create the secret, and sometimes they race on it, making the test flaky.
+	err := utils.Retry(
+		time.Second*3,
+		20,
+		func() (bool, error) {
+			_, err := cctx.VanClient.KubeClient.CoreV1().Secrets(cctx.Namespace).Get(
+				context.Background(),
+				secretName,
+				metav1.GetOptions{},
+			)
+			if err == nil {
+				return true, nil
+			}
+			if errors.IsNotFound(err) {
+				fmt.Printf("TLS Secret %q not yet created.  See #1208\n", secretName)
```

Comment:
Not sure about printing the number of a github issue in the logs, we can link this pull request to the issue for traceability, increase the priority, etc.

Reply:
@nluaces, I'm good to remove it, if you think that's more appropriate.  The reason I put it there, though, is to emphasize that that part of the code is temporary, and should be removed once #1208 is resolved.   I did the same thing on #1229.

Reply:
If you have done this for other pull requests, it is also fine for me, this is not really important. Thanks.